libs/designsystem/src/lib/components/card/directives/card-is-clickable.directive.ts

```diff
@@ -0,0 +1,16 @@
+import { Directive, OnInit } from '@angular/core';
+
+import { CardComponent } from '../card.component';
+
+@Directive({
+  // tslint:disable-next-line
+  selector: 'kirby-card[click]',
+})
+export class CardIsClickableDirective implements OnInit {
+  constructor(private card: CardComponent) {}
+
```

Comment:
```suggestion
@HostBinding('attr.role') role: string = 'button';
```
Consider adding this host binding on the directive level.

The reason we need role="button" is for this to be more accessible, so screen readers will announce and treat this element as a button. You can read more in that regard [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role).  

Now that we have the `role="button"` attribute, we could remove the class host binding, and use this for our 'hook' for styling instead. See my comment in the card.component.scss file.

Reply:
In order to make the improvements described in the above linked MDN docs with regards to the elements acting as buttons, we should make sure that the keyDown event for space and enter will also fire the click action. I believe this could be achieved with something like:
```suggestion
  @HostListener('keydown.space', ['$event'])
  @HostListener('keydown.enter', ['$event'])
  onKeydownHandler(evt: KeyboardEvent) {
    this.clickableElement.nativeElement.click(evt);
  }
```
This would require us to inject something like `private clickableElement: ElementRef` in the constructor.